HA.md

```diff
@@ -0,0 +1,21 @@
+# High Availability (HA)
+## Introduction
+When we create kubernetes cluster using kops we are able to create a multiple K8S masters and by doing so we can assure that in a case of 
+a master instance failure our K8S functionality won't damaged. 
+In scenario of a single master instance that fails our minions/pods will keep running but there will be no new pods schedualing, kubectl
+won't work (since the api-server will be down).
```

Comment:
We cannot deploy, if a node fails, failover will not be handled properly, and scheduled jobs will not be run :)